### ISSUE TITLE
build-neko.hxml fails with 4.0.0-rc.1

### ISSUE THREAD

**Issue by aliokan** [back2dos/travix#107]
When I try to run `haxelib git travix https://github.com/back2dos/travix && pushd . && cd $(haxelib config)travix/git && haxe build-neko.hxml && popd` with Haxe 4.0.0-rc.1
```
haxe build-neko.hxml
src/travix/commands/AuthCommand.hx:80: characters 14-28 : Type not found : HttpConnection
src/travix/commands/AuthCommand.hx:80: characters 14-28 : Type not found : HttpConnection
```
with `hx3compat`
```
haxe build-neko.hxml -lib hx3compat
src/travix/commands/AuthCommand.hx:83: characters 7-14 : haxe.remoting.HttpConnection has no field api
src/travix/commands/AuthCommand.hx:83: characters 7-14 : haxe.remoting.HttpConnection has no field api
```

**Comment by aliokan** [comment]
related to https://github.com/HaxeFoundation/hx3compat/issues/9

**Comment by kevinresol** [comment]
Yes need to use the resolve method

**Comment by kevinresol** [comment]
Closed by 99a6c9691410af6fafcfe6274ca463b53239d2ca
